### ISSUE TITLE
Digits and optional

### ISSUE THREAD

**Issue by shawnmclean** [CosticaPuntaru/redux-form-validation#16]
I'm setting a validation requirement that should be digits if there is any value but also optional.

My rules looks like:

```
{
  myProp: {
    required: false,
    digits: true,
    validateOnBlur: true
  }
}
```

This validates correctly for digits, but if the textbox is empty, the digits validation is still triggered. Is this a bug?
